t/14-isotovideo.t

```diff
@@ -394,6 +394,17 @@ subtest 'publish assets' => sub {
         is $return_code, 0, 'Successful handle_generated_assets' or die path(bmwqemu::STATE_FILE)->slurp;
         ok !-e $publish_asset, 'test.qcow2 does not exist';
     };
+
+    subtest 'prevent upload assets when publish_hdd is none with case-insensitive' => sub {
+        my @possible_values = qw(none None NONE);
+        for my $v (@possible_values) {
+            $bmwqemu::vars{PUBLISH_HDD_1} = $v;
+            $command_handler->test_completed(1);
+            my $return_code;
+            my $log = combined_from { $return_code = handle_generated_assets($command_handler, 1) };
+            like $log, qr/Asset upload is skipped for PUBLISH_HDD/, "Upload is skipped when PUBLISH_HDD_1 is $v";
+        }
+    };
```

Comment:
The corresponding test module for the file OpenQA/Isotovideo/Utils.pm is 20-openqa-isotovideo-utils.t . Can you consider moving your test code there?

Reply:
then that whole asset subtest should be moved there.
it wouldn't make sense to just move this single test